remote.go

```diff
@@ -595,6 +592,10 @@ func (o *Remote) UpdateFetchHead() bool {
 	return C.git_remote_update_fetchhead(o.ptr) > 0
 }
 
+func (o *Remote) PruneRefs() bool {
+	return C.git_remote_prune_refs(o.ptr) > 0
+}
```

Comment:
There are two methods, aren't there? one that retrieves the setting (this call), and another one that performs the prune, the one that you mention. Should I call the setting getter something different?